prompt.py

```diff
@@ -302,20 +332,49 @@ def do_export(self, arguments):
         item = get_arg(arguments)
 
         if item == 'wif':
+            if not self.Wallet:
```

Comment:
Here again might be easier to see whats going on by looking at the new version here: https://github.com/metachris/neo-python/blob/nep2-prompt/prompt.py#L334

I've also re-added the check for the user-supplied `address` as stated in `help`.